README.md

```diff
@@ -1,5 +1,5 @@
 # ![Favicon](target/apple-icon-57x57.png) Software Craft Website
-[![Build Status](https://travis-ci.org/softwarecrafters/website.svg?branch=master)](https://travis-ci.org/softwarecrafters/website) [![Deps](https://david-dm.org/softwarecrafters/website.svg)](https://david-dm.org/softwarecrafters/website) [![DevDeps](https://david-dm.org/softwarecrafters/website/dev-status.svg)](https://david-dm.org/softwarecrafters/website)
+[![Build Status](https://travis-ci.org/softwarecrafters/website.svg?branch=saga)](https://travis-ci.org/softwarecrafters/website) [![Deps](https://david-dm.org/softwarecrafters/website.svg)](https://david-dm.org/softwarecrafters/website) [![DevDeps](https://david-dm.org/softwarecrafters/website/dev-status.svg)](https://david-dm.org/softwarecrafters/website)
```

Comment:
Great suggestion! Did you want to name it `main` or `saga`?

Reply:
Right! I was using my goto alternative.